Somali-Dictionary/alphabetical-order/F words Qaamuus.txt

```diff
@@ -462,4 +462,168 @@ waxyaalaha foosha xun ee diintu reebtay.
+                          2. ld faafi.
+
+faaxid m.dh ld faafid.
+
+faaxin m.dh ld faafin¹.
+
+faaxis m.l ld faafin¹.
+
+faay¹ m.dh ld faayid.
+
+faay² f.mg1 (-yay, -yday) Caafimaad qabid.
+
+faayid m.f.dh eeg faay². ld faay¹.
+
+fac m.l (-cac, m.l) 1. Da'. 2. F. ah: dad isku da'
+ah oo isku waqti nool, jiil. 3. ld faacad.
+
+facaad m.dh ld fac (3).
+
+facaayil m.l.w eeg ficil.
+
+facaysad m.l ld facaysasho.
+
+facaysasho m.f.dh eeg facayso. ld facaysad,
+facaysi, filaysad, filaysasho, filaysi..
+
+facaysi m.l ld facaysasho.
+
+facayso f.g3 (-stay, -satay) Wax ama qof iskala
+fac dhigid. ld filayso.
+
+fackaallo m.dh Qof nuguur ama dabeecad xun
+oo aan lala dhaqmi karin.
+
+faclayn m.f.dh eeg faclee. ld facle (2).
+
+facle m.l 1. Orod afarqaadle ah oo aan xawli dheer lahayn oo ay ugaarta iyo xooluhu sameeyaan marka ay barwaaqaysanyihiin.
+          2. ld faclayn.
+
+faclee f.mg2 (-leeyay, -laysay) Facle samayn.
+
+facshar¹ m.l (-rro, m.dh) 1. Ceeb iyo sharafdhac weyn. 
+                          2. ld facsharid. 
+                          3. Kashifaad. ld faxshar¹.
+
+facshar² f.g1 (-ray, -rtay) Facshar ku samayn;
+ceebeyn. ld faxshar².
+
+facsharan f.mg1 (-rmay, -rantay; -rmi) Qof la
+facsharayay noqosho. ld faxsharan.
+
+facsharid m.f.dh eeg facshar². ld facshar1 (2),
+faxshar¹, faxsharid.
+
+facsharmid m.f.dh eeg facsharan. ld
+faxsharmid.
+
+fad m.l (-dad, m.l) 1. Daruur yar. 2. Wax yar
+oo meel ka soo go'ay. 3. ld fas.
+
+fadal m.l (-llo, m.dh) Raad noole oo dhulka
+qoday ama diisay.
+
+fadallayn m.f.dh eeg fadallee.
+
+fadallee f.mg2 (-leeyay, -laysay) Fadal samayn.
+ld fadli².
+
+fadaq m.l.kh F. ah: qof badnaa da' yar oo aan
+xasilloonayn.
+
+faddaaddo m.dh 1. Wax shaki leh ama aan caqligal ahayn. 
+               2. Dillaac dhaawac caallirasan. 
+               3. Marin laga bixiyey deyr ama qoqob. 
+               4. ld faddaro.
+
+faddarayn m.f.dh eeg faddaree.
+
+faddaraysan f.mg4 (-snaa, -snayd) Wax la
+faddareeyey ahaansho.
+
+faddaraysnaan m.f.dh eeg faddaraysan. ld
+faddaraysnaansho.
+
+faddaraysnaansho m.l/dh ld faddaraysnaan.
+
+faddare m.l.kh 1. F. ah: qof ku dhaqma wax aan diinta Islaamka iyo dhaqanku oggoleyn; nijaas. 
+               2. F. ah: wax nijaasi gaartay. ld faddaro.
+
+faddaree f.g2 (-reeyay, -raysay) Wax faddaro
+ka dhigid, wax baabi'in.
+
+faddaro m.dh.kh ld faddare.
+
+faddaroobid m.f.dh eeg faddarow². ld
+faddarow¹.
+
+faddarow¹ m.l ld faddaroobid.
+
+faddarow² f.mg1 (-oobay, -owday; -oobi)
+Wax, faddaro noqosho.
+
+faddhac¹ m.l.kh F. ah: go'idda wax cadad ka
+go'aan ee uu yeesho meelo daloola.
+
+faddhac² f.mg1 (-cay, -cday; -dhici) Wax,
+cadad meelo daloola u yeela ka go'id.
+
+faddhicid m.f.dh eeg faddhac².
+
+faddo m.dh 1. Meel dhulka kale in yar ka sarreysa, bannaan oo la dego waqtiga roobabawda. 
+           2. Taag.
+
+fadeexad m.dh ld fadeexo.
+
+fadeexayn m.f.dh eeg fadeexee.
+
+fadeexee f.g2 (-xeeyay, -xaysay) Cid fadeexo
+ku samayn; dacaayadeyn. 
+
 Completed page 310(287)
```

Comment:
Made a big mistake. It should be page 311(288)

Reply:
Have u checked how to revert commits and also what you can do is, you can copy all the text(pages) u did in f words and paste another txt file,then go back in time e.g when you began F words and paste all the pages or texts you did and commit a new commit with a message like "F words finished page 100(**) - page 110(**)" this is example but try. Good luck

Reply:
Thanks wll. Im in college today I will do it when I go home.